modules/AlphaWalletFoundation/AlphaWalletFoundation/TokenScriptClient/Models/XMLHandler.swift

```diff
@@ -759,7 +759,7 @@ final class ThreadSafe {
 }
 
 /// This class delegates all the functionality to a singleton of the actual XML parser. 1 for each contract. So we just parse the XML file 1 time only for each contract
-public class XMLHandler {
+public struct XMLHandler {
```

Comment:
Why do we do this?

Also it should be harmless because of `XMLHandler`'s properties are reference types, right?